### DISCUSSION TITLE
Slack link expired

### DISCUSSION THREAD

**Discussion by aretecarpe** [siderolabs/talos#9678] [Q&A]
Hello,

It appears that getting an invite via this link: https://slack.dev.talos-systems.io is expired. Could we update this please?

**Comment by smira** [comment]
This is being worked on as far as I know, but you can ask any questions here, in the Discussions, it would be much more valuable for the community.

**Comment by steverfrancis** [comment]
Updated

**Comment by dancmeyers** [comment]
I was about to post much the same thing, as it looks like the link on the website has expired again :( Is there any possibility of it being re-updated so I can get in?

I've also tried joining the community forum, which seems to be a Google Group, as a means to get an invite to the monthly Google Meet. But as far as I can see there's nothing in the conversations of the group and I've not been auto-sent an invite to the Meet call or anything?

**Reply by smira** [reply]
You can ask GitHub discussions (this forum) to ask questions as well, it works pretty well and stays as an archive of questions and answers

**Reply by rsmitty** [reply]
Slack link should be active again. Funny how generating a non-expiring link seems to expire :/
